graphql-dgs-codegen-gradle/src/main/kotlin/com/netflix/graphql/dgs/codegen/gradle/CodegenPlugin.kt

```diff
@@ -54,15 +54,37 @@ class CodegenPlugin : Plugin<Project> {
         project.configurations.create("dgsCodegen")
         project.configurations.findByName("dgsCodegen")?.isCanBeResolved = true
 
+        addDependencyLock(project, codegenExtension)
```

Comment:
This change makes sense to me, although I'll defer to @chali for his input on this.